pyqtgraph/graphicsItems/ROI.py

```diff
@@ -213,7 +213,7 @@ def angle(self):
         """Return the angle of the ROI in degrees."""
         return self.getState()['angle']
         
-    def setPos(self, pos, update=True, finish=True):
+    def setPos(self, pos, y=None, update=True, finish=True):
```

Comment:
What is the purpose of the y kwarg? Is there an interest to add some additional docstrings?